app/models/consistency.php

```diff
@@ -13,6 +13,29 @@
     $locale
 );
 
+$available_filters = [
+    'all'         => 'All products',
+    'firefox'     => 'Firefox for desktop and Android',
+    'seamonkey'   => 'SeaMonkey',
+    'thunderbird' => 'Thunderbird',
+];
+
+if (isset($_GET['filter'])) {
+    $selected_filter = isset($available_filters[$_GET['filter']])
+        ? $_GET['filter']
```

Comment:
I think I'd prefer that you pass this $_GET parameter through ```Utils::secureText()``` because even though it's just used in a ternary test later in ```Utils::getHtmlSelectOptions()```, if we later change that function and use the string directly that could cause a XSS.